Assets/LeapMotion/Core/Scripts/Attributes/QuickButtonAttribute.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Leap.Unity.Query;
+
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
+
+namespace Leap.Unity.Attributes {
+
+  public class QuickButtonAttribute : CombinablePropertyAttribute, IBeforeFieldAdditiveDrawer {
+
+    public const float PADDING_RIGHT = 12f;
+
+    public string label = "Quick Button";
+    public string methodOnPress = null;
+    public string tooltip = "";
+
+    public Type type;
+    public System.Reflection.MethodInfo method;
+
+    public QuickButtonAttribute(string buttonLabel, string methodOnPress, string tooltip = "") {
+      this.label = buttonLabel;
+      this.methodOnPress = methodOnPress;
+      this.tooltip = tooltip;
+    }
+
+    public float GetWidth() {
+      return GUI.skin.label.CalcSize(new GUIContent(label)).x + 12f + PADDING_RIGHT;
+    }
+
+#if UNITY_EDITOR
+    public void Draw(Rect rect, SerializedProperty property) {
+      
+      type = targets.Query().FirstOrDefault().GetType();
+      method = type.GetMethod(methodOnPress, System.Reflection.BindingFlags.Instance
```

Comment:
GetMethod can throw if there is an Ambiguous Match (more than one method with the same name), might be good to get all the methods and filter so you can throw a more descriptive error message.

https://msdn.microsoft.com/en-us/library/05eey4y9(v=vs.110).aspx